pkgs/tools/package-management/appimagekit/default.nix

```diff
@@ -11,29 +11,30 @@ let
   appimagekit_src = fetchFromGitHub {
     owner = "AppImage";
     repo = "AppImageKit";
-    rev = "b0859501df61cde198b54a317c03b41dbafc98b1";
-    sha256 = "0qqg79jw9w9rs8c2w3lla4kz62ihafrf7jm370pp1dl8y2i81jzg";
+    rev = "8bbf694455d00f48d835f56afaa1dabcd9178ba6";
+    sha256 = "sha256-pqg+joomC5CI9WdKP/h/XKPsruMgZEaIOjPLOqnNPZw=";
+    fetchSubmodules = true;
```

Comment:
Are the dependencies already in nixpkgs?

Reply:
If you mean the submodules, then no. There's a submodule `libappimage` which is not in nixpkgs. It's also fairly tightly integrated into AppImageKit (which references files in libappimage, not just the compiled result).

libappimage does have `gtest` as a dependency, which is in nixpkgs. Is there a way to only fetch the one submodule and not the other?

Reply:
> Is there a way to only fetch the one submodule and not the other?

Not that I am aware of.